### PR TITLE
Dev edit sentiment 0429

### PR DESCRIPTION
**Pull request by cjgronlund**
Developmental and copy edit to ML.NET sentiment analysis article

### CODE REVIEW

**Comment by cjgronlund** [comment]
@natke , @JRAlexander : Here's the complete dev edit. 

It's not a technical review. I did proofread to check whether I broke things.

@JRAlexander: Please check that I didn't break your tutorial. I didn't touch code and I didn't change the order of steps, so it should be OK. 

What I did:
* Add steps for readability.
* Simplify headings and remove duplication
* Align headings with Tutorial checklist
* Pare down some text.
* Combine some steps from H3s. The purpose of individual H3s wasn't always clear, and in some cases I made a call to combine and simplify.

File: docs/machine-learning/tutorials/sentiment-analysis.md [hunk 1]
```diff
@@ -95,115 +91,109 @@ The input dataset class, `SentimentData`, has a `string` for the comment (`Senti
...
+    |Service was very prompt.              |    1     |[0.39, 0, 0.75, …]    |
 
-Classification is a machine learning algorithm that uses data to **determine** the category, type, or class of an item or row of data and is frequently one of the following types:
+### Add a learning algorithm
 
-* Binary: either A or B.
-* Multiclass: multiple categories that can be predicted by using a single model.
-
-Because the website comments need to be classified as either positive or negative, you use the Binary Classification task.
+This app uses a classification algorithm that categorizes items or rows of data. The app categorizes website comments as either positive or negative, so use the binary classification task.
```
Comment (JRAlexander) [inline docs/machine-learning/tutorials/sentiment-analysis.md]: would classify be a better choice than categorize?

File: docs/machine-learning/tutorials/sentiment-analysis.md [hunk 2]
```diff
@@ -95,115 +91,109 @@ The input dataset class, `SentimentData`, has a `string` for the comment (`Senti
...
-The `FeaturizeText()` method in the previous code converts the text column (`SentimentText`) into a numeric key type `Features` column used by the machine learning algorithm and adds it as a new dataset column:
-
-|SentimentText                         |Sentiment |Features              |
-|--------------------------------------|----------|----------------------|
-|Waitress was a little slow in service.|    0     |[0.76, 0.65, 0.44, …] |
-|Crust is not good.                    |    0     |[0.98, 0.43, 0.54, …] |
-|Wow... Loved this place.              |    1     |[0.35, 0.73, 0.46, …] |
-|Service was very prompt.              |    1     |[0.39, 0, 0.75, …]    |
+### Extract and transform the data
+<!---The transform and featurization parts of this section seem like they'd be under "prepare data" above. A quirk of the ML.NET process? -->
```
Comment (JRAlexander) [inline docs/machine-learning/tutorials/sentiment-analysis.md]: yes.

File: docs/machine-learning/tutorials/sentiment-analysis.md [hunk 3]
```diff
@@ -95,115 +91,109 @@ The input dataset class, `SentimentData`, has a `string` for the comment (`Senti
...
 
-### Initialize variables in Main
+## Load the data
+Data in ML.NET is represented as an [IDataView class](xref:Microsoft.ML.IDataView). `IDataView` is a flexible, efficient way of describing tabular data (numeric and text). Data can be loaded from a text file or in real time (for example, SQL database or log files) to an `IDataView` object.
 
-Replace the `Console.WriteLine("Hello World!")` line in the `Main` method with the following code to declare and initialize the mlContext variable:
+You prepare the app, and then load data:
 
-[!code-csharp[CreateMLContext](~/samples/machine-learning/tutorials/SentimentAnalysis/Program.cs#CreateMLContext "Create the ML Context")]
+<!--- do we need to call out initialization explicitly, or could we just lump this together with loading data as I have-->
```
Comment (JRAlexander) [inline docs/machine-learning/tutorials/sentiment-analysis.md]: I think this is ok.

**Review by JRAlexander — COMMENTED** [review]
I like the edits. Just one clarification.

File: docs/machine-learning/tutorials/sentiment-analysis.md (hunk 1 above)
Comment (cjgronlund) [reply docs/machine-learning/tutorials/sentiment-analysis.md]: It's best not to use essentially the same word to define a word.

File: docs/machine-learning/tutorials/sentiment-analysis.md (hunk 3 above)
Comment (cjgronlund) [reply docs/machine-learning/tutorials/sentiment-analysis.md]: I'll remove my comments. Thanks for these replies, John. :)

**Review by JRAlexander — APPROVED** [review]
LGTM! Thanks, @cjgronlund, for the thoughtful insights!

**Pull request merged**